reports/distributions/contrib.yaml

```diff
@@ -67,6 +67,7 @@ components:
         - health_check
         - host_observer
         - http_forwarder
+        - http_forwarder
```

Comment:
I don't think Contrib distro will include two `http_forwarder` extensions?

Reply:
Oh these reports are generated from the metadata. I wonder if metadata should be updated?

Reply:
no I don't want to change the type of the component.